pkg/utils/utils.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	"crypto/sha512"
+	"fmt"
+)
+
+const (
+	maxChainLength = 29
+	chainPrefix    = "CNI-"
+	prefixLength   = len(chainPrefix)
+)
+
+// Generates a chain name to be used with iptables.
+// Ensures that the generated chain name is less than
+// maxChainLength chars in length
+func FormatChainName(name string, id string) string {
+	chainBytes := sha512.Sum512([]byte(name + id))
```

Comment:
https://play.golang.org/p/HeGo6Ig66D

With your function,
```
	name := "Hello "
	id := "world!"
```
will have the same hash as:
```
	name := "Hello worl"
	id := "d!"
```